stix_shifter_modules/vectra/stix_transmission/connector.py

```diff
@@ -223,6 +223,16 @@ def get_results_data(self, response_dict):
 
             detection_type = record.get('detection_type', '')
 
+            # adding new field for confidence field mapping
+            if 'certainty' in record:
+                record['certainty_score'] = record['certainty']
+
+            # converting 0 to 1 for severity and confidence
+            if record.get('certainty_score') == 0:
+                record['certainty_score'] = 1
+            if record.get('threat') == 0:
+                record['threat'] = 1
```

Comment:
put some detail comments about this change. for example: the reason for altering the value

Reply:
Updated with comments